staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -180,8 +180,10 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.DurationVar(&s.StorageConfig.DBMetricPollInterval, "etcd-db-metric-poll-interval", s.StorageConfig.DBMetricPollInterval,
 		"The interval of requests to poll etcd and update metric. 0 disables the metric collection")
-}
 
+	fs.Int64Var(&s.StorageConfig.LeaseManagerConfig.ReuseDurationSeconds, "lease-reuse-duration-seconds", s.StorageConfig.LeaseManagerConfig.ReuseDurationSeconds,
+		"The time in seconds that each lease is reused. A lower value could avoid large number of objects reusing the same lease. Notice that a too small value may cause performance problems at storage layer.")
+}
```

Comment:
nit: empty line after } and no empty line before this flag

Reply:
Added empty line after }

Please note that there is only one line before this flag (line. 183), which matches other flags in this file.

The "new line" in 184 is simply github's line wrapping of 184 line into two lines (you can use "..." > "View file" to see the full file).